src/sentry/static/sentry/app/views/settings/organization/members/organizationMembersView.jsx

```diff
@@ -34,6 +45,7 @@ class OrganizationMembersView extends AsyncView {
       members: [],
       invited: new Map(),
       accessRequestBusy: new Map(),
+      searchQuery: idx(this.props, _ => _.location.query.query),
```

Comment:
I would make sure this defaults to empty string if `location.query.query` is undefined (otherwise we get React warnings about the input being uncontrolled).